reference/snapshot.md

```diff
@@ -0,0 +1,279 @@
+---
+Title: OpenEBS Volume Snapshot
+Sidebar: Volume snapshot
+---
+
+
+**Volume snapshots** are copies of a persistent volume claims (PVCs) at a specific point in time. They can be used to restore a volume to a previous state or create a new volume. Mayastor provides support for industry standard copy-on-write (COW) snapshots, which is a popular methodology for taking snapshots by keeping track of only those blocks that have changed.
```

Comment:
Should be "copies of a persistent volume". I think I'd suggested PVC earlier, but it seems incorrect.